brain-marks/Settings/Views/whatsnew.md

```diff
@@ -0,0 +1,5 @@
+**v1.2**
```

Comment:
This is fine, if you don't want to change it, but I'd prefer for the version numbers to be like this `# v1.2`, which would show them as a heading rather than just bold